web/src/games/checkers/game.ts

```diff
@@ -170,10 +127,10 @@ export function getValidMoves(G: IG, playerID: string, jumping?: ICheckerPieceWi
 }
 
 export function move(G: IG, ctx: Ctx, from: ICoord, to: ICoord): IG | string {
-  const fromCoord = new Coord(from.x, from.y);
-  const toCoord = new Coord(to.x, to.y);
-  const indexFrom = fromCoord.toIndex();
-  const indexTo = toCoord.toIndex();
+  const fromCoord = coord(from.x, from.y);
+  const toCoord = coord(to.x, to.y);
```

Comment:
These 2 initial conversions should also be unnecessary (see my previous comment on "fromICoord").

Reply:
Done.